pkg/azure/plugin_integration_test.go

```diff
@@ -32,7 +32,7 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func createVM(ctx context.Context, server *azurePluginServer, subscriptionId string, resourceGroupName string, namespace string, location string, name string) (*paragliderpb.CreateResourceResponse, error) {
+func createParagliderVM(ctx context.Context, server *azurePluginServer, subscriptionId string, resourceGroupName string, namespace string, location string, name string) (*paragliderpb.CreateResourceResponse, error) {
```

Comment:
Any reason this was renamed? If this clashes with another name, then a better name might be `createTestVM`. We don't have a concept known as a "Paraglider VM", so the method might be misleading.

Also, I prefer to keep names short as Go is already verbose.

Reply:
I was initially thinking of creating a function that returns a VM not created within Paraglider's vnet. However, I decided against that, so I would rename to as before